src/components/tickets/ticket/create-ticket/create-ticket.tsx

```diff
@@ -67,159 +72,144 @@ export class CreateNewTicket extends React.Component<
-                  min={0}
-                  max={1}
-                  name="priority"
-                  onChange={(event) => this.handleChange(event)}
-                  step={0.01}
-                  type="range"
-                  value={this.state.priority}
-                />
-                <p>
-                  Current priority:
-                  <ColorIndicator percentage={this.state.priority} />
-                  {this.state.priority}
-                </p>
-                <Divider />
-
-                <div>
-                  <p>
-                    <strong>Set reminder:</strong>
-                  </p>
-                  <Checkbox
-                    toggle
-                    label="Include a reminder"
-                    checked={this.state.setReminder}
-                    onChange={() => {
-                      const flippedState = !this.state.setReminder
-                      this.setState({ setReminder: flippedState })
-                      if (this.state.setReminder !== true) {
-                        const [date, time] = formatDateTime(new Date())
-                        this.setState({
-                          remindDate: date,
-                          remindTime: time,
-                          remindMessage: this.state.description,
+                      .then(() => {
+                        this.props.closeModal()
+                        this.props.showNotification({
+                          header: 'Success!',
+                          message: 'Created a new ticket.',
+                          type: 'green',
                         })
-                      } else {
-                        this.setState({
-                          remindDate: null,
-                          remindTime: null,
-                          remindMessage: null,
+                      })
+                      .catch((error) => {
+                        this.props.showNotification({
+                          header: 'Error!',
+                          message: error.message,
+                          type: 'red',
                         })
-                      }
-                    }}
+                        throw error
+                      })
+                  }}
+                >
+                  <Label htmlFor={'description'}>Description:</Label>
+                  <Form.TextArea
+                    row={4}
+                    col={20}
+                    name="description"
+                    placeholder="Type in a description"
+                    value={this.state.description}
+                    onChange={(e) => this.handleChange(e)}
+                  />
+                  <Divider />
+                  <Label htmlFor="assignedTo">Assign to:</Label>
+                  <Form.Dropdown
+                    name="assignedTo"
+                    placeholder={
+                      teamOptions[this.state.currentUser.iEXOptionsIndex].text
+                    }
```

Comment:
Isn't placeholder something different to the default value?
```suggestion
                    defaultValue={ this.state.currentUser.iEXOptionsIndex > -1 ? teamOptions[this.state.currentUser.iEXOptionsIndex].text : null} 
```

Reply:
defaultValue only populates dropdown field on first render, meaning when getting it from a async query, it populates it with null. placeholder-attribute populates regardless. It stills assigns in the background meaning its only estethic. I could find a workaround